### PR TITLE
Selectors 

### PR DESCRIPTION
**Pull request by gmxer**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206849254647614

### CODE REVIEW

File: src/context/SyntheticsStateContext/selectors/chartSelectors.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// const { chartToken, availableChartTokens } = useMemo(() => {
+//     if (!fromTokenAddress || !toTokenAddress) {
+//       return {};
+//     }
```
Comment (midas-myth) [inline src/context/SyntheticsStateContext/selectors/chartSelectors.ts:4]: 🧹

File: src/domain/synthetics/fees/useRebatesInfo.ts [hunk 2]
```diff
@@ -32,11 +32,11 @@ export type RebatesInfoResult = {
   claimablePositionPriceImpactFees: RebateInfoItem[];
 };
 
-export function useRebatesInfo(chainId: number): RebatesInfoResult {
+export function useRebatesInfoRequest(chainId: number, enabled: boolean): RebatesInfoResult {
```
Comment (midas-myth) [inline src/domain/synthetics/fees/useRebatesInfo.ts:35]: why is enabled needed?

File: src/pages/Dashboard/DashboardV2.css [hunk 3]
```diff
@@ -20,7 +20,6 @@
 .token-table-wrapper {
   border-radius: 0.4rem;
   background: #17182c;
-  padding: 0 1.5rem;
```
Comment (midas-myth) [inline src/pages/Dashboard/DashboardV2.css:23]: why was this nesessary?

File: src/components/Synthetics/AccruedPositionPriceImpactRebateModal/AccruedPositionPriceImpactRebateModal.tsx [hunk 4]
```diff
@@ -2,75 +2,54 @@ import { Trans, t } from "@lingui/macro";
...
-        acc[index].push(rebateItem);
-      } else {
-        groupedMarkets[key] = acc.length;
-        acc.push([rebateItem]);
-      }
 
-      return acc;
-    }, [] as RebateInfoItem[][]);
-  }, [accruedPositionPriceImpactFees]);
+export const AccruedPositionPriceImpactRebateModal = memo(
```
[the remark this answers is not in the document]
Comment (gmxer) [reply src/components/Synthetics/AccruedPositionPriceImpactRebateModal/AccruedPositionPriceImpactRebateModal.tsx:18]: Ideally all components should have it to not render if props are not changed. But I forget this rule and don't promote it at all for now

also
image

File: src/domain/synthetics/fees/useRebatesInfo.ts (hunk 2 above)
Comment (gmxer) [reply src/domain/synthetics/fees/useRebatesInfo.ts:35]: this is ugly way to not fetch unused stuff in SyntheticsStateContextProvider

image

But I don't have a better idea for now

File: src/pages/Dashboard/DashboardV2.css (hunk 3 above)
Comment (gmxer) [reply src/pages/Dashboard/DashboardV2.css:23]: in production this rule is overrided by AppCard rule in App.scss. I changed the order of css imports in this PR somehow. So this line should be just removed

Here's the subtask regarding it https://app.asana.com/0/0/1207061230803208/f

**Review by midas-myth — APPROVED** [review]

**Pull request merged**